### PR TITLE
docs: textcat tutorial and small doc fixes

### LINKED ISSUE #5035: [DOCS] Textcat Tutorial
## Which page or section is this issue related to?

- configure dataset
- upload data
- annotate data (SetFit zero-shot but mention LLMs)
- train model (SetFit)
- update suggestions

### PR DESCRIPTION
**Pull request by sdiazlor**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Closes #5035 

**Type of change**

(Remember to title the PR according to the type of change)

- [x] Documentation update

**How Has This Been Tested**

(Please describe the tests that you ran to verify your changes.)

`mkdocs serve`

### CODE REVIEW

**Review by davidberenstein1957 — COMMENTED** [review]
As discussed you can update the mkdocs to be excluded from the pre-commit checks. Maybe double check the new redirect URLs.

There are some typos in the text, maybe copy-paste it back and forth between Grammarly?

Maybe mention brief alternatives to consider in sections. For example "Add suggestions with a zero-shot SetFit model" might be a bit made a bit more general. "Add initial model suggestion" and/or it could mention that SetFit is one approach but people can consider using other techniques like LLMs or zero-shot models? 

Maybe the for zero-shot training it is not super clear what is happening? Perhaps briefly mention that it is inferring labels based on data entries or something. 

Maybe we can reference the, how to use the UI guide in the "evaluate with argilla" section?

Perhaps we can explain why we are using only 8 samples per label.

For the conclusion, maybe mention this is the base for an iterative progress about data quality etc. 

https://huggingface.co/TaylorAI/bge-micro-v2 is a good and very small baseline model, perhaps we can use that one?

**Review by davidberenstein1957 — COMMENTED** [review]
@sdiazlor thanks for this. Did you also check if Argilla-Python does not get mentioned in the page descriptions anymore?

File: argilla/docs/tutorials/text_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,529 @@
+{
```
Comment (davidberenstein1957) [inline argilla/docs/tutorials/text_classification.ipynb:1]: <p>The next step is to add suggestions to the dataset. In our case, we will generate them using a zero-shot SetFit model. However, you can use a framework or technique of your choice.</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5055/discussion/#comment-1646356490'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='23'/>

Comment (davidberenstein1957) [inline argilla/docs/tutorials/text_classification.ipynb:1]: <p>The <code>id</code> will always need to be provided as it is the records' identifier to update a record and avoid creating a new one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5055/discussion/#comment-1646356492'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='34'/>

Comment (davidberenstein1957) [inline argilla/docs/tutorials/text_classification.ipynb:1]: <p>Voilà! We have added the suggestions to the dataset, and they will appear in the UI marked with a ✨. </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5055/discussion/#comment-1646356494'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='36'/>

Comment (davidberenstein1957) [inline argilla/docs/tutorials/text_classification.ipynb:1]: <p>We can train the model using our previous function, but this time with a high-quality human-annotated training set.</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5055/discussion/#comment-1646356495'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='46'/>

Comment (davidberenstein1957) [inline argilla/docs/tutorials/text_classification.ipynb:1]: <p>remaining means, not annotated?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5055/discussion/#comment-1646356497'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='48'/>

**Comment by sdiazlor** [comment]
@davidberenstein1957 I'll check. site_url: https://argilla-io.github.io/argilla-python should be changed too?

File: argilla/docs/tutorials/text_classification.ipynb (hunk 1 above)
Comment (sdiazlor) [reply argilla/docs/tutorials/text_classification.ipynb:1]: <p>yes</p>

**Review by davidberenstein1957 — APPROVED** [review]

**Pull request merged**